package.json

```diff
@@ -109,7 +109,9 @@
     "test:reuse:service": "npm run test:reuse:service:rest && npm run test:reuse:service:odata",
     "test:reuse": "npm run test:reuse:common && npm run test:reuse:nonUi5 && npm run test:reuse:ui5 && npm run test:reuse:util && npm run test:reuse:service:odata",
     "test:reuse:helper:elementHighlight": "npx wdio ./test/reuse/helper/elementHighlight/test.elementHighlight.conf.js",
-    "test": "npm run test:authenticator:staticLogin && npm run test:core && npm run test:reuse"
+    "test": "npm run test:authenticator:staticLogin && npm run test:core && npm run test:reuse",
+    "android:local": "npx wdio wdio.local.android.device.conf.js",
```

Comment:
Probably it's better to follow convention used in the other npm commands here: `test:mobile:android:local`

Reply:
Also, the command doesn't run, I guess it's because config file route is incorrect. You would either need to add a file (and then place it to the same folder as the other config), or remove the command